CONTRIBUTING.md

```diff
@@ -78,6 +78,51 @@ In particular, please try to write in the past tense (e.g. "Added a new feature"
 
 Thanks for your help with this.
 
+### How we version packages
+
+Keystone follows the semver model of {major}.{minor}.{patch}. Version numbers are the first and most obvious way we have of communicating changes to our users, so it's important we convey consistent meaning with them, and strike a careful balance between releasing often vs. overloading consumers with package updates.
+
+Generally, versions should be interpreted like:
+
+- `major` means a breaking change to the public API of a package, and/or a meaningful change to the internal behaviour
+- `minor` means we added a feature to the package, which is backwards compatible with the current major version
+- `patch` means a bug has been fixed in the package
```

Comment:
I prefer referring to these as `breaking` / `feature addition` / `bug fix` instead of major, minor, patch. It helps convey the meaning much better imo.

Reply:
Isn't that basically what it says? I'm confused.

Reply:
Sorry, I meant the first word of each dotpoint:

- `breaking` means a breaking change ...
- `feature addition` means we added a feature ...
- `bug fix` means a bug has ...